sprintQuovantis/sprintParam/urls.py

```diff
@@ -4,4 +4,6 @@
 urlpatterns = [
     path('sprint', views.Sprint.as_view(), name='sprint'),
     path('param_sprint', views.ParameterOfSprint.as_view(), name='param_sprint'),
+    path('user_vote', views.UserVote.as_view(), name='user_vote'),
+    path('result', views.Result.as_view(), name='result'),
```

Comment:
what is this path for? results of a sprint? I guess you haven't built that API yet? and shouldn't it have {sprint_id} in the url path? How will the api user provide what sprint_id he wants the results for?